test/e2e/common_test.go

```diff
@@ -0,0 +1,126 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types"
```

Comment:
is there similar api for podman?

Reply:
Yes. The API we already use in device-worker: "github.com/containers/podman/v3/pkg/bindings/containers"
But we use docker here, because github workflow is using docker by default, and also because I had some issues executing systemd in podman, but works fine in docker.